lambdas/call-ticf-cri/src/test/java/uk/gov/di/ipv/core/callticfcri/service/TicfCriServiceTest.java

```diff
@@ -50,44 +42,47 @@
 import static uk.gov.di.ipv.core.callticfcri.service.TicfCriService.TRUSTMARK;
 import static uk.gov.di.ipv.core.callticfcri.service.TicfCriService.X_API_KEY_HEADER;
 import static uk.gov.di.ipv.core.library.domain.CriConstants.TICF_CRI;
-import static uk.gov.di.ipv.core.library.fixtures.TestFixtures.EC_PRIVATE_KEY_JWK;
-import static uk.gov.di.ipv.core.library.fixtures.TestFixtures.M1A_ADDRESS_VC;
-import static uk.gov.di.ipv.core.library.fixtures.TestFixtures.M1A_EXPERIAN_FRAUD_VC;
-import static uk.gov.di.ipv.core.library.fixtures.TestFixtures.M1B_DCMAW_VC;
-import static uk.gov.di.ipv.core.library.fixtures.TestFixtures.VC_ADDRESS;
+import static uk.gov.di.ipv.core.library.fixtures.VcFixtures.M1A_ADDRESS_VC;
+import static uk.gov.di.ipv.core.library.fixtures.VcFixtures.M1A_EXPERIAN_FRAUD_VC;
+import static uk.gov.di.ipv.core.library.fixtures.VcFixtures.M1B_DCMAW_VC;
+import static uk.gov.di.ipv.core.library.fixtures.VcFixtures.vcAddressOne;
 
 @ExtendWith(MockitoExtension.class)
 class TicfCriServiceTest {
-
     private static final ObjectMapper objectMapper = new ObjectMapper();
     private static final ClientOAuthSessionItem clientSessionItem =
             ClientOAuthSessionItem.builder()
                     .vtr(List.of("vtr-value"))
                     .userId("a-user-id")
                     .govukSigninJourneyId("a-govuk-journey-id")
                     .build();
-    private static final List<String> credentials =
-            List.of(M1B_DCMAW_VC, M1A_EXPERIAN_FRAUD_VC, M1A_ADDRESS_VC);
+    private static List<String> credentials;
     // the VC in this response is unimportant as we're mocking the
     // validator - we just need something that can be parsed
-    private static final TicfCriDto ticfCriResponse =
-            new TicfCriDto(
-                    List.of("vtr-value"),
-                    Vot.P2,
-                    TRUSTMARK,
-                    "a-user-id",
-                    "a-govuk-journey-id",
-                    List.of(VC_ADDRESS));
-
+    private static TicfCriDto ticfCriResponse;
     private IpvSessionItem ipvSessionItem;
     private RestCriConfig ticfCriConfig;
-
     @Mock private ConfigService mockConfigService;
     @Mock private HttpClient mockHttpClient;
     @Mock private VerifiableCredentialJwtValidator mockVerifiableCredentialJwtValidator;
     @Mock private HttpResponse<String> mockHttpResponse;
     @Captor private ArgumentCaptor<HttpRequest> requestCaptor;
     @InjectMocks private TicfCriService ticfCriService;
+    private static String VC_ADDRESS;
+
+    @BeforeAll
+    static void setVcs() throws Exception {
+        credentials = List.of(M1B_DCMAW_VC, M1A_EXPERIAN_FRAUD_VC, M1A_ADDRESS_VC);
```

Comment:
Does this still need to be done statically?

It's not massively important, but it feels like we're a little inconsistent. If you'd like to change it, it might also be easier to merge this and then fix up after :)

Reply:
Oops, no I missed this - I'll revert this to initialising this inline